packages/ketcher-core/src/application/render/restruct/resgroup.ts

```diff
@@ -14,25 +14,50 @@
  * limitations under the License.
  ***************************************************************************/
 
-import { Box2Abs, FunctionalGroup, Pile, SGroup, Vec2 } from 'domain/entities';
-
+import {
+  Box2Abs,
+  FunctionalGroup,
+  Pile,
+  SGroup,
+  Vec2,
+  Bond,
+} from 'domain/entities';
 import { SgContexts } from 'application/editor/shared/constants';
-import { LayerMap } from './generalEnumTypes';
 import ReDataSGroupData from './redatasgroupdata';
+import ReStruct from './restruct';
+import { Render } from '../raphaelRender';
+import { LayerMap } from './generalEnumTypes';
 import ReObject from './reobject';
 import { Scale } from 'domain/helpers';
 import draw from '../draw';
 import util from '../util';
 import { tfx } from 'utilities';
+import { RaphaelPaper } from 'raphael';
+import { RenderOptions } from '../render.types';
+interface SGroupdrawBracketsOptions {
+  set: any;
+  render: Render;
+  sgroup: SGroup;
+  crossBonds: { [key: number]: Array<Bond> };
+  atomSet: Pile;
+  bracketBox: Box2Abs;
+  d: Vec2;
+  lowerIndexText?: string | null;
```

Comment:
What do you mean here by `index`?

Reply:
Index here as far as I understood from code and S-Groups behaviour is this 
<img width="504" alt="Screenshot 2023-08-14 at 20 42 34" src="https://github.com/epam/ketcher/assets/109075831/c5c5c6db-0e4a-48ff-84c1-7001c5a10cc2">